pe.go

```diff
@@ -0,0 +1,198 @@
+package fdeutil
+
+import (
+	"debug/pe"
+	"encoding/binary"
+	"fmt"
+	"io"
+)
+
+// readDataDirectories accepts a io.ReadSeeker pointing to data directories in the PE file,
+// its size and number of data directories as seen in optional header.
+// It parses the given size of bytes and returns given number of data directories.
+func readDataDirectories(r io.ReadSeeker, sz uint16, n uint32) ([]pe.DataDirectory, error) {
+	ddSz := binary.Size(pe.DataDirectory{})
+	if uint32(sz) != n*uint32(ddSz) {
+		return nil, fmt.Errorf("size of data directories(%d) is inconsistent with number of data "+
```

Comment:
(nitpick^99) Just a style thing but the snapd code does not do line breaks after 80chrs and its also slightly uncommon in the go community to do this.

Reply:
Ah, I don't mind changing that, but perhaps outside of this PR as all of the code is like it.